Enable centos-gluster*-test repo

soumyakoduri opened the pull request:
https://review.gluster.org/18524 introduces new API to set lk_owner which is needed by nfs-ganesha from here on. Since it shall take a while before 3.12.3 is released and the API is available, Niels suggested that we can trigger and use test-build till then. Hence enabling the gluster test repo.

soumyakoduri committed:
Enable centos-gluster*-test repo

https://review.gluster.org/18524 introduces new API to set lk_owner which is needed by nfs-ganesha from here on. Since it shall take a while before 3.12.3 is released and the API is available, Niels suggested that we can trigger and use test-build till then. Hence enabling the gluster test repo.

nixpanic commented on common-scripts/basic-gluster.sh:
```diff
@@ -30,6 +30,7 @@ set -x
 
 # enable repositories
 yum -y install centos-release-gluster yum-utils
+yum --enablerepo=centos-gluster*-test install glusterfs-api-devel
```
Please do this further below at
~~~
# install NFS-Ganesha build dependencies
~~~

soumyakoduri replied:
Done. Please check the same.

soumyakoduri committed:
Enable centos-gluster*-test repo 

https://review.gluster.org/18524 introduces new API to set lk_owner which is needed by nfs-ganesha from here on. Since it shall take a while before 3.12.3 is released and the API is available, Niels suggested that we can trigger and use test-build till then. Hence enabling the gluster test repo.

nixpanic commented:
Note that the builds are a work in progress, and not available yet. The glusterfs-3.12.2-2 RPMs will show up at https://buildlogs.centos.org/centos/7/storage/x86_64/gluster-3.12/ later today.
